R/uniDimBayesianCalcCoeff.R

```diff
@@ -306,17 +307,24 @@
     ciValue <- options[["credibleIntervalValueScale"]]
 
     if (is.null(out[["samp"]])) {
-      startProgressbar(model[["progressbarLength"]] %/% 500 + 1)
-      out[["samp"]] <- coda::mcmc(apply(model[["gibbsSamp"]], MARGIN = c(1, 2), Bayesrel:::glbOnArray_custom,
-                                        callback = progressbarTick))
+
+      dd <- dim(model[["gibbsSamp"]])
+      out[["samp"]] <- matrix(0, dd[1], dd[2])
+
+      startProgressbar(dd[1] * 3)
+      for (i in 1:dd[1]) {
```

Comment:
```suggestion
      for (i in seq_len(dd[1])) {
```
I also wouldn't mind if you use `nrow` and `ncol` directly, although that is slightly more verbose. Up to you.